pkgs/development/python-modules/pyobjc-core/default.nix

```
@@ -0,0 +1,111 @@
+# inpired by https://github.com/input-output-hk/lace/blob/0fd166666cc454171811b307050d0815452bea82/nix/lace-blockchain-services/internal/any-darwin.nix#L339
+{
+  lib,
+  pkgs,
+  stdenv,
+  buildPythonPackage,
+  fetchPypi,
+  xcodebuild,
+  cctools,
+  darwin,
+}:
+let
+  appleSDK = darwin.apple_sdk_11_0;
+  inherit (appleSDK) MacOSX-SDK objc4;
+  darwinFrameworks = appleSDK.frameworks;
+
+  apple_libffi = stdenv.mkDerivation {
+    pname = "apple-libffi";
+    inherit (MacOSX-SDK) version;
+    dontUnpack = true;
+    installPhase = ''
+      mkdir -p $out/include $out/lib
+      cp -r ${MacOSX-SDK}/usr/include/ffi $out/include/
+      cp -r ${MacOSX-SDK}/usr/lib/libffi.* $out/lib/
+    '';
+  };
+in
+
+buildPythonPackage rec {
+  pname = "pyobjc_core";
```

```suggestion
  pname = "pyobjc-core";
```
> Attribute names in python-packages.nix as well as pnames should match the library's name on PyPI, but be normalized according to [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names). This means that characters should be converted to lowercase and . and _ should be replaced by a single - (foo-bar-baz instead of Foo__Bar.baz). If necessary, pname has to be given a different value within fetchPypi.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines